content/collections/docs/white-labeling.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'White Labeling'
+intro: 'White Labeling allows you to customize the logo, visible name, and basic theme of the CMS throughout the control panel.'
+template: page
+stage: 'Needs Polish & Humor'
+updated_by: 3a60f79d-8381-4def-a970-5df62f0f5d56
+updated_at: 1612297554
+id: 5bd9426f-23cf-4196-9848-471dff67f5ea
+---
+
+## Configuration
+Your White Label options are avialble in `config/statamic/cp.php` or through their corresponding .env variables.
```

Comment:
Small little thing, available is spelled wrong here 🙂

Reply:
Thanks!